.github/workflows/versioning.yml

```
@@ -0,0 +1,45 @@
+    inputs:
+      update_stable:
+        description: "Update stable tag?"
+        required: true
+        default: "false"
+
+jobs:
+  versioning:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)